.travis.yml

```diff
@@ -20,6 +20,10 @@ env:
       XW1hzORAtSpTgTKkQwel5gRMDy6SotzeSRsVV2jQCn46VIMx8G/J5nOI+ImL
       yeoH12PhCR0h39dM7mq8TYJo5DHwvbotI5nQhpMruSt8eMFbym8nGiqQh806
       fSJXkxmQ4MAjUdNFDIirBHhdZme8q3PueFzJ+5odFMvPGn/aITQ=
+script:
+  - npm run lint
+  - npm run lint-spec
+  - npm test
```

Comment:
I would put `lint` and `lint-spec` to `pretest` script so that they're called automatically before `npm test`. And it will be consistent with the current behavior where `npm test` does all the necessary checks.

Reply:
When tests in real browsers will be added to Travis, I would like to have separate test script for each browser (for example `npm test --browsers Firefox` and `npm test --browsers Chrome` instead of `npm test --browsers Firefox,Chrome`) to avoid mixing logs from different browsers (for better readability).
Adding `lint` and `lint-spec` to `pretest` script will trigger linting for each test run, which makes no sense.

From the other hand, having linting scripts in`pretest` would be handy for local development.

What solution you think is better?

Reply:
@ghybs thanks for your comment!

>Having the lint triggered for each browser in Travis does not harm much.

It's true, but if it's possible to avoid this duplication I'd like to do it :)
I really like solution you proposed! Will update PR!